src/config/database/migrations/20200802012857-create-access-tokens.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('access_tokens', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
```

Comment:
Codacy found an issue: [Identifier 'user_id' is not in camel case.](https://app.codacy.com/manual/lucas.assuncao.p/node-bloodboiler-sequelized/pullRequest?prid=5950393)